libwild/src/args.rs

```diff
@@ -638,6 +657,30 @@ pub(crate) fn parse<F: Fn() -> I, S: AsRef<str>, I: Iterator<Item = S>>(input: F
             args.allow_shlib_undefined = true;
         } else if long_arg_eq("no-allow-shlib-undefined") {
             args.allow_shlib_undefined = false;
+        } else if let Some(rest) = get_option_value("unresolved-symbols") {
+            match rest.as_str() {
+                "report-all" => {
+                    args.no_undefined = true;
+                    args.unresolved_symbols = UnresolvedSymbols::ReportAll;
+                }
+                "ignore-in-shared-libs" => {
+                    args.no_undefined = true;
```

Comment:
I'm a little unsure why we need this boolean in addition to `args.unresolved_symbols`. It kind of feels like it's doing the same thing, but perhaps I'm missing something.